docs/en/latest/admin-api.md

```diff
@@ -1312,6 +1312,12 @@ Plugin resource request address: /apisix/admin/plugins/{plugin_name}
 | ------ | ----------------------------------- | ------------ | ---------------------------------------------- |
 | GET    | /apisix/admin/plugins/list          | NULL         | Fetches a list of all Plugins.                 |
 | GET    | /apisix/admin/plugins/{plugin_name} | NULL         | Fetches the specified Plugin by `plugin_name`. |
+| GET         | /apisix/admin/plugins?all=true      | NULL         | Get all properties of all plugins. |
```

Comment:
Add deprecate note for `all=true`.

@Revolyssup btw, does `apisix/admin/plugins` do what `/apisix/admin/plugins?all=true` do after the change? Or is the subsystem URL param _required_ for the`/apisix/admin/plugins` endpoint

Reply:
@kayx23 The only way to return all results is with `all=true`. Even with `subsystem`, plugin name has to be provided. The mechanism of getting all plugins at once was deprecated with the usage of all=true.